lib/utils/escape-filter-value.test.js

```diff
@@ -83,3 +83,9 @@ tap.test('leaves encoded characters intact', async t => {
     t.equal(result, expected[i])
   }
 })
+
+tap.test('parse binary data with end byte >0xc0', async t => {
+  const input = Buffer.from([0x30, 0x41, 0x62, 0xd2])
+  const expected = '0Ab\\d2'
+  t.equal(escapeFilterValue(input), expected)
+})
```

Comment:
```suggestion
tap.test('handle start of two byte utf-8 character at end of buffer', async t => {
  const input = Buffer.from([0x7e, 0xc2])
  const expected = '~\\dc2'
  t.equal(escapeFilterValue(input), expected)
})
```

We are also missing a test for the three byte case.

Reply:
OK, I will split into 2 more specific test